### ISSUE TITLE
Integer vs Decimal

### ISSUE THREAD

**Issue by jcpny1** [learn-co-students/flatiron-store-project-v-000#233]
Is it best practice to use integers for currency instead of decimal?

**Comment by hcarnes** [comment]
No, I used decimal.

**Comment by jcpny1** [comment]
I did as well. I'm trying to remember why I posted this issue. I think it was because the only indication of a datatype in the source repository was in the seeds.rb file, where it says 'price: Faker::Number.number(4)', which would produce integers. As opposed to .decimal(4,2), for example, which would produce decimal numbers.

**Comment by daniellecroxton** [comment]
I tend to agree. I got a little stuck on the Store Index price tests. I ended up changing my price data type to decimal, the seed data to `price: Faker::Number.decimal(2)`, and the store_home_feature_spec.rb to  `expect(page).to have_content "$#{item.price}"`

That made more sense to me.
